charts/steward/crds/pipelineruns.yaml

```diff
@@ -80,7 +80,7 @@ spec:
                 default: run
               "timeout": ###
                 type: string
-                pattern: '([0-9]+h)?([0-9]+m)?([0-9]+s)?'
+                pattern: '^([0-9]+h)?([0-9]+m)?([0-9]+s)?$'
```

Comment:
looks correct. However I would expect also a note in the changelog. If you faced some issues because of this pattern, it would be also useful to mention this in the changelog/PR.

Reply:
The pattern was introduced newly with the prior pull which is not yet released.